Github Issue Reader/Views/IssuesCollectionVC.swift

```diff
@@ -11,6 +11,12 @@ private let reuseIdentifier = "Cell"
 
 class IssuesCollectionVC: UICollectionViewController, UICollectionViewDelegateFlowLayout {
     
+    var cellHeight: CGFloat = 100
+    
+    func title() {
+        navigationItem.title = "Issues"
+    }
```

Comment:
This function is never called so it never assigns the title of the `navigationItem` to "Issues". It's also not an override that changes an existing function.